File: components/apacheWebServer.lua
#!/usr/bin/env lua

apacheWebServer = {}

function apacheWebServer.preInstall()
  print "apacheWebServer.preInstall"
end

function apacheWebServer.install()
  print "apacheWebServer.install"
end

function apacheWebServer.postInstall()
  print "apacheWebServer.postInstall"
end

return apacheWebServer

File: components/apacheWebServer.rb
#!/usr/bin/env jruby

remoteRequire 'components/gnuBuild'

class ApacheWebServer < GnuBuild
  
  def preInstall
    puts("apacheWebServer.preInstall")
  end

  def install
    puts("apacheWebServer.install")
  end

  def postInstall()
    puts("apacheWebServer.postInstall")
  end
  
  def configure()
    puts("apacheWebServer.configure")
  end
  
end
